### ISSUE TITLE
Binance backfill doesn't work

### ISSUE THREAD

**Issue by Orwi53** [DeviaVir/zenbot#1030]
### Describe the problem
Binance backfill doesn't work

The error can be reproduced, by running, for example:

    zenbot backfill binance.ETH-BTC --days 30

### Source code / logs

![zenbotbug1](https://user-images.githubusercontent.com/35005865/34472352-b70ad778-ef60-11e7-9418-65e94c8f0d6d.png)

**Comment by ramene** [comment]
Please see #513, the issue is also discussed #1011

I'm able to see:
```

binance.XRP-BTC saved 257 trades 30 days left
.......................
binance.XRP-BTC saved 9279 trades 29 days left
.......................
binance.XRP-BTC saved 9423 trades 28 days left
.......................
binance.XRP-BTC saved 13374 trades 27 days left
...........
```

**Comment by Orwi53** [comment]
The fix you suggested on Issue #513 worked! Can't we push it?

**Comment by DeviaVir** [comment]
https://github.com/DeviaVir/zenbot/pull/1027
